x-pack/plugins/security_solution/server/lib/sourcerer/routes/index.ts

```diff
@@ -90,6 +82,7 @@ export const createSourcererDataViewRoute = (
             }
           }
         } else if (patternListAsTitle !== siemDataViewTitle) {
+          siemDataView = await dataViewService.get(dataViewId);
```

Comment:
We can get to this line only if `siemDataView` is not null, so I didn't understand the logic, why we need to reassign the new value fetched from `dataViewService` to `siemDataView`, is there any purpose for this?

Reply:
yes, unfortunately to make the `dataViewService.updateSavedObject` call we need to pass the entire data view object including fields 😢

Reply:
@stephmilovic is this still true? From the docs it says `Update part of an data view. Only the specified fields are updated in the data view. Unspecified fields stay as they are persisted.`. Are we just looking to update title here?

Reply:
Thanks for the follow-up Steph! That stinks...